docs/user/pretrained_network_experiment.md

```diff
@@ -0,0 +1,56 @@
+# Use a pre-trained network to start a new experiment
+
+In this section, we will provide a guide on how to use a pre-trained network to start a new experiment in garage. In practice, it is common for people to pretrain a network on a very large dataset and use the pre-trained network to perform the task of interest in reinforcement learning.
+
+This tutorial will walk you through starting an experiment with a pre-trained network using two of the most common deep learning frameworks, Tensorflow and Pytorch.
+
+## Set up the environment
+
+First, we will start by setting up the environment for the experiment. We will train a `VPG algorithm` as an example.
+
+We will simply setup the environment by initializing the garage environment `GarageEnv` and the `LocalRunner` instance.
+
+```python
+def vpg_pendulum(ctxt=None, seed=1):
+    set_seed(seed)
+    env = GarageEnv(env_name='InvertedDoublePendulum-v2')
+
+    runner = LocalRunner(ctxt)
+
+    ....
+```
+
+## Load the pre-trained model
```

Comment:
we don't actually use these mechanisms for saving models, though maybe we should. instead, we pickle the entire model object (`Policy`, `QFunction`, etc.)

check out the examples folder